### ISSUE TITLE
Configure number_separator for minimum length for either side of decimal

### ISSUE THREAD

**Issue by bjarkehs** [realm/SwiftLint#1200]
I ran into this while working on a project using coordinates. While I would like the minimum length of 5, it should apply to either side of the decimal.

So taking the example from #1109 I find `1000.01` more readable than `1_000.01` or `10.0003` more readable than `10.000_3`

Maybe even be able to ignore after the decimal.

Is this something others would be interested in as well? Because then I'd look into making a PR.

**Comment by marcelofabri** [comment]
I think that makes sense. Feel free to send a PR! 💯

**Comment by bjarkehs** [comment]
I just realized while looking at the code that this is actually how it works. This is an oversight/a case of the mondays.

I have another idea, and please tell me if this is not really useful or too specific. I'm also not entirely sure on the naming/api.

But let's say I want to have a minimum length of 5 for everything before decimal, so as #1109 suggests: `10000` becomes `10_000` but `1000` stays the same. But at the same time I want my decimals to be split at a later point. So my code could look like this:

```swift
let number = 10_000
let coord = 56.00529
let coord2 = 57.045_423_2
```

The configuration could be something like
```yml
number_separator:
  minimum_length: 5
  minimum_fraction_length: 7
```

**Comment by marcelofabri** [comment]
That was what I thought you ment in the original issue 😅

**Comment by bjarkehs** [comment]
Haha, cool. I'll do that then.

I was thinking of ways to specify it better, but any other would break current API I guess.

This should be sufficient anyway.
